.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/javascript/"
+    schedule:
+      interval: "weekly"
```

Comment:
is there a longer interval we can set? weekly is way too frequent, IMO. I think monthly is a good compromise, what do you think?

Reply:
Sure, monthly could work. Also, and perhaps more importantly, if we wish to cut down on notifications and PR volumes we could set the `group` parameter so that multiple dependency updates are bundled in one PR.

In the short term though, I'd actually quite like to experiment with the dependabot PRs and see if the existing ones are closed & updated appropriately. So, it could be helpful to start with frequent updates initially whilst we validate it's working, with a view to switching to reducing the PR volume once it's settled down. Does that work for you?